Makefile

```diff
@@ -20,7 +20,7 @@ build: verify
 	go build -o $(BINDIR)/ovirt-csi-driver -ldflags '-X version.Version=$(REV)' github.com/ovirt/csi-driver/cmd/ovirt-csi-driver
 
 .PHONY: verify
-verify: fmt vet test
```

Comment:
These tests assume the node and controller plugin are in the same process (pod), which is no longer the case.  I will create an issue to fix them and re-enable the tests.